src/xlsx/extraction/chart_extractor.ts

```diff
@@ -53,6 +61,8 @@ export class XlsxChartExtractor extends XlsxBaseExtractor {
             ],
           stacked: barChartGrouping === "stacked",
           fontColor: "000000",
+          horizontal: chartDirection === "bar",
+          isDoughnut: chartHoleSize > 0,
```

Comment:
FYI we'll support custom hole size with this task https://www.odoo.com/odoo/2328/tasks/4735861

Might be worth updating this, depending on what task is merged first :)